yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/SimpleActivityDetailsTest.java

```diff
@@ -77,21 +80,23 @@ public void shouldReturnReceiptId() {
     
     @Test
     public void shouldReturnBase64SelfieIfSelfieSet() {
```

Comment:
I find this test a little clearer if its...
```
@Test
    public void shouldReturnBase64SelfieIfSelfieSet() {
        Attribute attribute = new Attribute("selfie", new JpegAttributeValue(SOME_SELFIE_BYTES), null);
        SimpleProfile testProfile = new SimpleProfile(asList(attribute));

        SimpleActivityDetails result = new SimpleActivityDetails(USER_ID, testProfile, APP_PROFILE, TIMESTAMP, RECEIPT_ID);

        String expected = "data:image/jpeg;base64," + Base64.toBase64String(SOME_SELFIE_BYTES);
        assertEquals(expected, result.getBase64Selfie());
    }
```

Reply:
:+1: